quickjs.c

```diff
@@ -34561,7 +34561,7 @@ static JSString *JS_ReadString(BCReaderState *s)
     }
 #ifdef DUMP_READ_OBJECT
     if (check_dump_flag(s->ctx->rt, DUMP_READ_OBJECT)) {
-        bc_read_trace(s, "");  // hex dump and indentation
+        bc_read_trace(s, "%s", ""); // hex dump and indentation
```

Comment:
Ha, nice! Can you please remove the silencer flag I added to CMake then?

Reply:
sure